ui/src/app/tasks/task-apps/task-apps.component.ts

```diff
@@ -0,0 +1,60 @@
+import { Component, OnInit } from '@angular/core';
+import { Subscription } from 'rxjs/Subscription';
+import { ToastyService, ToastyConfig, ToastOptions, ToastData } from 'ng2-toasty';
+import { Router, ActivatedRoute, Params } from '@angular/router';
+import { AppRegistration } from '../../apps/model/app-registration';
+import { Page } from '../../shared/model/page';
+import { AppsService } from '../../apps/apps.service';
```

Comment:
Yeah, tasks were later added to show only task apps meaning wrong structure is coming from there. Btw, I starter using reload flag on that service method every time as I noticed that local caching on apps service returns stale data if i.e you restart scdf server and import different apps.